src/tests/multihost/alltests/test_proxy_provider_krb_auth.py

```diff
@@ -0,0 +1,122 @@
+""" Automation of proxy provider suite
+
+:requirement: IDM-SSSD-REQ : Proxy Provider
+:casecomponent: sssd
+:subsystemteam: sst_idm_sssd
+:upstream: yes
+"""
+from __future__ import print_function
+import pytest
+import paramiko
+import subprocess
+from sssd.testlib.common.utils import sssdTools, SSHClient
+
+
+def execute_cmd(multihost, command):
+    """ Execute command on client """
+    cmd = multihost.client[0].run_command(command)
+    return cmd
+
+
+@pytest.mark.usefixtures('setup_sssd_krb',
+                         'create_posix_usersgroups',
+                         'sssdproxyldap_test')
+@pytest.mark.tier1_3
+class TestProxy(object):
+    """
+    This is test case class for ldap proxy suite
+    """
+    def test_proxy_lookup(self, multihost, backupsssdconf):
+        """
+        :title: Proxy lookup and kerberos auth
+        :id: 5c4c55b8-0cac-47d0-aa23-d057b790e18e
+        :steps:
+          1. Check ldap user has access to client machine
+          2. Proxy lookup and kerberos auth
+          3. Change expired password
+        :expectedresults:
+          1. Should succeed
+          2. Should succeed
+          3. Should succeed
+        """
+        tools = sssdTools(multihost.client[0])
+        client_e = multihost.client[0].ip
+        tools.clear_sssd_cache()
+        ssh1 = SSHClient(client_e, username="foo2@example1",
+                         password="Secret123")
+        ssh1.close()
+        assert "home/foo2:/bin/bash" in execute_cmd(multihost,
+                                                    "getent -s "
+                                                    "ldap passwd "
+                                                    "foo2").stdout_text
+        # proxy lookup and kerberos auth
+        tools.clear_sssd_cache()
+        execute_cmd(multihost, "> /var/log/message")
+        assert "home/foo2:/bin/bash" in execute_cmd(multihost,
+                                                    "getent -s "
+                                                    "ldap passwd "
+                                                    "foo2").stdout_text
+        ssh1 = SSHClient(client_e, username="foo2@example1",
+                         password="Secret123")
+        ssh1.close()
+        with pytest.raises(subprocess.CalledProcessError):
+            execute_cmd(multihost, 'grep "undefined symbol" '
```

Comment:
I do not understand what this `grep` should check?

Reply:
This is from this line:  

https://gitlab.cee.redhat.com/sssd/sssd-qe/-/blob/RHEL8.6/client/proxy_provider/krb_auth/runtest#L91

Reply:
Hi,

but even there it is not clear why the grep was added. According to the git history it was coming from an even older version. The grep would indicate a very special reason for a failure, if the configured proxy pam_module is not installed. Imo this is not very useful information and the grep can be removed.

Additionally, if the grep is remove the test is basically calling `getent passwd` and `ssh` twice, only in different order, I think one run can be remove.

bye,
Sumit